sys/include/net/gnrc/sixlowpan/config.h

```diff
@@ -301,6 +301,84 @@ extern "C" {
 #ifndef CONFIG_GNRC_SIXLOWPAN_SFR_DG_RETRIES
 #define CONFIG_GNRC_SIXLOWPAN_SFR_DG_RETRIES            0U
 #endif
+
+/**
+ * @brief   The denominator for the factor for when to mark ECN on `netif` input
+ *          queue state
+ *
+ * When `gnrc_sixlowpan_frag_sfr_ecn_if_in` is compiled in, nodes will set
+ * the ECN bit of an RFRAG header when the input queue of the outgoing `netif`
+ * is filled by
+ * @ref CONFIG_GNRC_SIXLOWPAN_SFR_ECN_IF_IN_NUM / @ref CONFIG_GNRC_SIXLOWPAN_SFR_ECN_IF_IN_DEN
```

Comment:
When would one modify these?

Reply:
For the experiments we'd done for [the paper](https://ieeexplore.ieee.org/document/9580895) we played around a bit with different thresholds, but in the end the loss due to LLNs was so high, that it did not make much difference. However, I am not yet indifferent against removing this config option. Maybe somebody wants to redo those experiments with a more stable MAC (e.g. with #18156).